### DISCUSSION TITLE
Cypress testing with Quasar components

### DISCUSSION THREAD

**Discussion by devjared1** [quasarframework/quasar#14381] [General - Components / Directives / etc]
I just recently added Quasar to my Vue3/Vite project via the Vite plugin with
```
npm install quasar @quasar/extras
npm install -D @quasar/vite-plugin sass@1.32.12
```

Before adding Quasar, I was just using standard html elements and custom css. I was able to add ` data-cy` attributes to the elements so that I could test with Cypress component testing using
```
npx cypress open-ct
```

However, now I have upated some of my components to use Quasar components, like `q-btn` or `q-input`. I was hoping that I would still be able to add an attribute to the Quasar components with `data-cy`, but now whenever I run the component test runner I get a bunch or weird errors.

Is there a way currently to run Cypress tests on components that use Quasar components?

**Comment by MilosPaunovic** [comment]
This could be your starting point:
https://quasar.dev/quasar-cli-webpack/testing-and-auditing

**Reply by devjared1** [reply]
That seems to be using the Quasar CLI, which I am not using. I am using Vite
